### ISSUE TITLE
State legislative map coloring adhere with style guide

### ISSUE THREAD

**Issue by hjohns12** [PrincetonUniversity/PEC-map#9]
The style guide for colors is here: https://github.com/Princeton-Election-Consortium/pec-style-guide

Is it possible to update the state leg. colors using colors from this palette?

**Comment by mike-iezzi** [comment]
I tried using those colors and several other iterations, I thought the colors I chose conveyed the relative values best and were not to be confused with D/R colors.  Jacob seemed to agree when I sent it to him, but of course we could use that palette if you think that would be better.

**Comment by hjohns12** [comment]
Yeah, I see what you mean about choosing colors that don't indicate D/R as a challenge. The ones in use don't have a very strong contrast against the base map so they don't stand out very much (except for the hot pink districts).

@indraneel what do you think? How important is it that we stick with the color palette?

**Comment by mike-iezzi** [comment]
I agree about the contrasts, I did that intentionally because I wanted only the highest 2 power range districts to visually draw attention based off my conversation with Jacob.  When I used higher contrast colors for each value range, it was visually challenging to tell what color actually is the highest value, without looking at the legend--I think this type of coloring is a bit more obvious which regions are the highest.  

Again, I'm fine with them being anything else, that was just my thinking.
